entrypoint.sh

```diff
@@ -228,6 +228,16 @@ function seiso_lint() {
 			linter["$key"]=$value
 		done < <(echo "$line" | jq -r 'to_entries|map("\(.key)=\(.value|tojson)")|.[]')
 
+		if [[ ${AUTO_FIX-} == "true" ]]; then
```

Comment:
Why not :- see https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
Done